src/main/java/com/nha/abdm/wrapper/hip/hrp/dataTransfer/HealthInformationService.java

```diff
@@ -198,7 +197,7 @@ private ResponseEntity<GenericResponse> pushHealthInformation(
       HealthInformationBundle healthInformationBundle, String consentId)
       throws InvalidAlgorithmParameterException, NoSuchAlgorithmException, InvalidKeySpecException,
           NoSuchProviderException, InvalidKeyException {
-    log.info(healthInformationBundle);
+    log.debug("HealthInformationBundle : " + healthInformationBundle.toString());
```

Comment:
this can throw NPE

Reply:
Removed .toString() to avoid nullPointerException.